src/main/java/com/minecolonies/coremod/colony/requestsystem/init/StandardFactoryControllerInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.minecolonies.coremod.colony.requestsystem.init;
+
+import com.minecolonies.api.colony.requestsystem.StandardFactoryController;
+import com.minecolonies.api.colony.requestsystem.token.StandardTokenFactory;
+import com.minecolonies.coremod.colony.requestsystem.locations.EntityLocation;
+import com.minecolonies.coremod.colony.requestsystem.locations.StaticLocation;
+import com.minecolonies.coremod.colony.requestsystem.requests.StandardRequestFactories;
+
+/**
+ * Initializer for the {@link StandardFactoryControllerInitializer}
+ */
+public final class StandardFactoryControllerInitializer
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)